6 comments to article

Jord000 opened the pull request:
improvements to previous component build and now comments by article shown on individual article page. 
nav bar added to go back and increased error handling and UI for general navigation through the app

nickkagame commented on MVC/errors.js:
```diff
@@ -1,3 +1,10 @@
+exports.sqlError = (err, req, res, next) => {
+  if (err.code === '22003') {
```
Does this error get triggered by your code?  I'm expecting to see a custom error in your model.

nickkagame commented on __tests__/app.test.js:
```diff
@@ -78,4 +78,20 @@ describe('GET:200 /GET/api/articles/:article_id', () => {
   })
+  test('should return correct error when wrong article_id provided', () => {
+    return request(app)
+      .get('/api/articles/999999999999')
```
try and make this number smaller and then see what error you get back!

nickkagame reviewed (changes requested):
Hi Jordan,  Nice work so far! But I'm a bit confused as this code as far as I can see shouldn't give you a 404, as you should require a custom error in the model.  Please have a look at the comments and then see if you can make the changes and resubmit please.  Esp.  try a small article ID for your 404 test!

Jord000 committed:
fixed error in console on main feed removed console logs

Jord000 committed:
truncate text and grids improved on main feed button added for more detail

Jord000 committed:
error to login
styling for all pages so far to help navigate
page for individual article but no return button
dynamic endpoints created

Jord000 committed:
nav bar to get back to feed added

Jord000 committed:
set so without login myfeed defaults to login
nav bar works better now and is setup for further detail later

Jord000 committed:
set up for comments on article page

Jord000 committed:
comments added by article

Jord000 committed:
error removed

Jord000 committed:
improved routing method using link to get parametric endpoint

Jord000 committed:
Merge branch '5-individual-article' into 6-comments-to-article

Jord000 committed:
merging branch 5
